build/update-crdb-certs.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
This script seems to be a subset of `apply-certs.sh` -- can we factor out this functionality from `apply-certs.sh` and have `apply-certs.sh` call this script so that we aren't repeating ourselves?